pkgs/tools/audio/beets/builtin-plugins.nix

```diff
@@ -10,6 +10,11 @@
   ...
 }:
 {
+  _typing = {
+    testPaths = [ ];
+    enable = false;
+    builtin = false;
+  };
```

Comment:
New in `unstable`, disabling by default/marking as not built-in in previous version of `beets`